pylint_odoo/test/main.py

```diff
@@ -62,6 +62,7 @@
     'attribute-string-redundant': 33,
     'renamed-field-parameter': 2,
     'deprecated-data-xml-node': 5,
+    'resource-not-exist': 4,
```

Comment:
@pedrobaeza 
I deleted the not referenced file `test.yml`
Now only have 3 resources what not found
@moylop260 Thanks for answering